openassessment/assessment/test/test_staff.py

```diff
@@ -0,0 +1,348 @@
+# coding=utf-8
+import datetime
+import pytz
+import copy
+import mock
+
+from django.db import DatabaseError, IntegrityError
+from django.utils import timezone
+from ddt import ddt, file_data
+from nose.tools import raises
+
+from openassessment.test_utils import CacheResetTest
+from openassessment.assessment.api import staff as staff_api
+from openassessment.assessment.api.self import create_assessment as self_assess
+from openassessment.assessment.models import (
+    Assessment, AssessmentPart, AssessmentFeedback, AssessmentFeedbackOption,
+    PeerWorkflow, PeerWorkflowItem
+)
+from openassessment.assessment.errors import StaffAssessmentRequestError, StaffAssessmentInternalError
+from openassessment.workflow import api as workflow_api
+from submissions import api as sub_api
+
+STUDENT_ITEM = dict(
+    student_id="Tim",
+    course_id="Demo_Course",
+    item_id="item_one",
+    item_type="Peer_Submission",
+)
+
+ANSWER_ONE = u"this is my answer!"
+
+# Possible points: 14
+RUBRIC_DICT = {
```

Comment:
Yeah, that would be much cleaner. I'll get working on that.

@dianakhuang - I'll handle all the feedback on the tests, since I wrote them initially.